ansible/roles/debops.tinyproxy/defaults/main.yml

```diff
@@ -0,0 +1,501 @@
+        comment: |
+          DisableViaHeader: When this is set to yes, Tinyproxy does NOT add
+          the Via header to the requests. This virtually puts Tinyproxy into
+          stealth mode. Note that RFC 2616 requires proxies to set the Via
+          header, so by enabling this option, you break compliance.
+          Don't disable the Via header unless you know what you are doing...
+        raw: |
+          DisableViaHeader Yes
+        state: 'comment'
+
+      - name: 'Filter'
+        comment: |
+          Filter: This allows you to specify the location of the filter file.
+        raw: |
+          Filter "/etc/tinyproxy/filter"
+        state: 'comment'
+
+      - name: 'FilterURLs'
+        comment: |
+          FilterURLs: Filter based on URLs rather than domains.
+        raw: |
+          FilterURLs On
+        state: 'comment'
+
+      - name: 'FilterExtended'
+        comment: |
+          FilterExtended: Use POSIX Extended regular expressions rather than
+          basic.
+        raw: |
+          FilterExtended On
+        state: 'comment'
+
+      - name: 'FilterCaseSensitive'
+        comment: |
+          FilterCaseSensitive: Use case sensitive regular expressions.
+        raw: |
+          FilterCaseSensitive On
+        state: 'comment'
+
+      - name: 'FilterDefaultDeny'
+        comment: |
+          FilterDefaultDeny: Change the default policy of the filtering system.
+          If this directive is commented out, or is set to "No" then the default
+          policy is to allow everything which is not specifically denied by the
+          filter file.
+
+          However, by setting this directive to "Yes" the default policy becomes
+          to deny everything which is _not_ specifically allowed by the filter
+          file.
+        raw: |
+          FilterDefaultDeny Yes
+        state: 'comment'
+
+      - name: 'Anonymous'
+        comment: |
+          Anonymous: If an Anonymous keyword is present, then anonymous proxying
+          is enabled.  The headers listed are allowed through, while all others
+          are denied. If no Anonymous keyword is present, then all headers are
+          allowed through.  You must include quotes around the headers.
+
+          Most sites require cookies to be enabled for them to work correctly, so
+          you will need to allow Cookies through if you access those sites.
+        raw: |
+          Anonymous "Host"
+          Anonymous "Authorization"
+          Anonymous "Cookie"
+        state: 'comment'
+
+      - name: 'ConnectPort'
+        comment: |
+          ConnectPort: This is a list of ports allowed by tinyproxy when the
+          CONNECT method is used.  To disable the CONNECT method altogether, set
+          the value to 0.  If no ConnectPort line is found, all ports are
+          allowed (which is not very secure.)
+
+          The following two ports are used by SSL.
+        raw: |
+          ConnectPort 443
+          ConnectPort 563
```

Comment:
Port 563 is `nntps`, NNTP over SSL. I wonder how did it get included here by the upstream...